hudi-client/src/main/java/org/apache/hudi/table/HoodieCopyOnWriteTable.java

```diff
@@ -18,17 +18,13 @@
 
 package org.apache.hudi.table;
 
+import static org.apache.hudi.common.util.HoodieWriteClientUtils.averageBytesPerRecord;
+
 import com.google.common.hash.Hashing;
 import java.io.IOException;
 import java.io.Serializable;
 import java.nio.charset.StandardCharsets;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Could you split this import into seperate imports?

Reply:
sure.